README.md

```diff
@@ -26,7 +26,7 @@ module "security_hub_collector_runner" {
   permissions_boundary = // optional, defaults to ""
 
   schedule_task_expression  = "cron(30 9 * * ? *)"
-  scheduled_task_enabled    = // optional, defaults to true
+  scheduled_task_enabled    = // optional, defaults to ENABLED
```

Comment:
I think you need to update the variable name to `scheduled_task_state` here and below

Reply:
Oh right, updated.